### DISCUSSION TITLE
Who was the Roman equivalent of the Greek god Hephaestus?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#262] [Galaxy Trivia] [answered]
**Hint:** 🌋

**Comment by live627** [comment]
Vulcan

**Comment by thuongtruong1009** [comment, marked as answer]
The Roman equivalent of the Greek god Hephaestus is Vulcan. Like Hephaestus, Vulcan was associated with fire and metalworking, and was regarded as the patron of blacksmiths, craftsmen, and artisans. In Roman mythology, Vulcan was often depicted as a skilled artisan who crafted weapons, armor, and other metal objects for the gods and heroes. He was also believed to be responsible for volcanic eruptions and other natural phenomena associated with fire and heat.

**Comment by RexRowan** [comment]
Vulcan

**Comment by darko5r** [comment]
Vulcan
